CONTRIBUTING.md

```diff
@@ -92,7 +92,15 @@ Review fixes should be pushed on separate commits for easier viewing on GitHub (
 
 ## Release TODO list
 
-1. Merge the changes from `develop` to `master` and push `master`.
+1. Create release PR from `develop` into `master`.
+
+1. Verify that all checks have passed.
+
+1. Merge develop into master and push to remote repository using console:
+
+         git checkout master
```

Comment:
From my experience, it happens pretty often that ppl have their `develop`/`master` branches NOT up to date with remote when doing this FF merge... so I'd also add `git pull`s on both branches